recipes/kim/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "kim-api" %}
+{% set version = "2.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openkim/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d433ad8edebe29311da6f362ff65830785cb880b2152735bf62676a77e19bbb4
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - vim
+  host:
+    - wget
```

Comment:
What is wget being used for?

Reply:
They use it to download additional models from the web https://github.com/openkim/kim-api/blob/master/INSTALL#L93

Reply:
It looks like a runtime requirement rather than a host one? I think it's only used when calling `kim-api-collections-management install`?

Reply:
Yes, you are right.